src/python_framework_v02/troute/nhd_io.py

```diff
@@ -189,12 +193,20 @@ def get_ql_from_wrf_hydro_mf(qlat_files, index_col="feature_id", value_col="q_la
         preprocess=drop_all_coords,
         # parallel=True,
     ) as ds:
-        ql = pd.DataFrame(
-            ds[value_col].values.T,
-            index=ds[index_col].values,
-            columns=ds.time.values,
-            # dtype=float,
-        )
+        try:
+            ql = pd.DataFrame(
+                ds[value_col].values.T,
+                index=ds[index_col].values[0],
+                columns=ds.time.values,
+                # dtype=float,
+            )
```

Comment:
Dig deeper into this difference in the qlateral files -- is there an intentional difference between the output from the RnR method and the files we are receiving from the later model runs?

The except statement works with the more current files. The try statement works with the RnR output -- it fails quickly on a type mismatch, so it is not causing a significant performance issue.